client/keys/add.go

```diff
@@ -36,6 +36,41 @@ const (
 	DefaultKeyPass = "12345678"
 )
 
+func addKeyScriptingCommand() *cobra.Command {
```

Comment:
is this change necessary or related to the PR?

Reply:
Ah cc @jackzampolin probably unnecessary

Reply:
Its necessary to make the demo work seamlessly and I would argue for this feature regardless of IBC. I'm happy to pull it out into a separate PR, but I think it would be best if we merge as is and open an issue to review and finalize this CLI testing utility.

Reply:
Can you open an issue for it?  Which were the old steps for the demo to work without this change?